### ISSUE TITLE
Noisy Hepatic vessel prediction on training data

### ISSUE THREAD

**Issue by Amritpal-001** [bwittmann/vesselFM#3]
Using the pretrained weights and the hepatic_vessel dataset that was used for training in this paper, I get random very noisy mask for the liver vessel segmentation.

Image used is hepticvessel_001.nii.gz from task 08 heaptic_vessel segmentation from MSD dataset

![Image](https://github.com/user-attachments/assets/b14a93a3-9e4f-4fee-9a2b-10d12a0567ff).

Please advise on how to fix it.

Code used:

```
conda create -n vesselfm python=3.9
conda activate vesselfm
pip install -e .

# for image segmentation
python vesselfm/seg/inference.py
```

**Comment by Amritpal-001** [comment]
I haven’t found any solution yet. Waiting for the author’s reply here.

**Comment by alfredtorres** [comment]
Hi Amritpal,

After taking a closer look at the repository, I realized that there is actually data preprocessing code included: [convert_MSD.py](https://github.com/bwittmann/vesselFM/blob/main/vesselfm/d_real/dataset_conversion/convert_MSD.py) .
I followed the preprocessing steps and conducted a zero-shot test. The result on MSD was 0.2884, which is quite close to the original paper's result of 0.2969.

I hope this information might also be useful for you.

**Comment by bwittmann** [comment]
_'Using the pretrained weights and the hepatic_vessel dataset that was used for training in this paper [...]'_

Please also note that we never trained on the MSD8 dataset. We solely used it to evaluate the performance of our model
